crates/api_common/src/utils.rs

```diff
@@ -38,6 +44,8 @@ use rosetta_i18n::{Language, LanguageId};
 use tracing::warn;
 use url::{ParseError, Url};
 
+pub static AUTH_COOKIE_NAME: &str = "jwt";
```

Comment:
Why change the cookie name back to jwt, just to make it more backward compatible?

Reply:
To be honest its up to you and @SleeplessOne1917 which way its easier to handle the cookie. So let me know if you prefer to use the same name (`jwt`, means you need to mark the existing cookie as secure and httponly), or rename it (then you can set a new cookie and dont worry about the old one). Though if you use header for auth it doesnt matter at all.

Reply:
Doesn't matter to me. Thoughts @SleeplessOne1917 ?

Reply:
imo the hasJwtCookie and setJwtCookie functions should be removed from lemmy-ui, and with the cookie being named jwt here it shouldn't require any further changes there (I think). lemmy-ui shouldn't need to worry about the cookie at all

Reply:
I'm with phiresky on this one.

Reply:
Actually the breaking changes post says that the cookie is called `auth`. So I renamed it back to that. 

https://lemmy.ml/post/5711722

Reply:
i guess it doesn't matter, lemmy-ui should still be able to remove their cookie logic regardless of what the new cookie is named